### ISSUE TITLE
Error: 'ggwithinstats' is not an exported object from 'namespace:ggstatsplot'

### ISSUE THREAD

**Issue by cailiangliang765** [IndrajeetPatil/ggstatsplot#215]
Please briefly describe your problem and what output you expect. If you have a question, please don't use this form. Instead, ask on <https://stackoverflow.com/> or <https://community.rstudio.com/>.

Please include a minimal reproducible example (AKA a reprex). If you've never heard of a [reprex](http://reprex.tidyverse.org/) before, start by reading <https://www.tidyverse.org/help/#reprex>.

---

i run the demo on your page.

```r
# insert reprex here
library(ggstatsplot)
library(WRS2)
ggstatsplot::ggwithinstats(
+     data = WRS2::WineTasting,
+     x = Wine,
+     y = Taste,
+     sort = "descending", # ordering groups along the x-axis based on
+     sort.fun = median, # values of `y` variable
+     pairwise.comparisons = TRUE,
+     pairwise.display = "s",
+     pairwise.annotation = "p",
+     title = "Wine tasting",
+     caption = "Source: `WRS2` R package",
+     ggtheme = ggthemes::theme_fivethirtyeight(),
+     ggstatsplot.layer = FALSE,
+     messages = FALSE
+ )
```
Error: 'ggwithinstats' is not an exported object from 'namespace:ggstatsplot'

**Comment by IndrajeetPatil** [comment]
This is a new function still under development and not present in the `CRAN` version. You will, therefore, have to download the development version of the package with the following command-
```
remotes::install_github(repo = "IndrajeetPatil/ggstatsplot")
```

**Comment by cailiangliang765** [comment]
> This is a new function still under development and not present in the `CRAN` version. You will, therefore, have to download the development version of the package with the following command-
> 
> ```
> remotes::install_github(repo = "IndrajeetPatil/ggstatsplot")
> ```

@IndrajeetPatil Many thanks for your reply and help.

**Comment by Solpero33** [comment]
Hello, i try to instal as you say and i have this error:
`Installing package into ‘/Users/labepara/Library/R/3.5/library’
(as ‘lib’ is unspecified)
* installing *source* package ‘ggstatsplot’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  namespace ‘boot’ 1.3-20 is already loaded, but >= 1.3.21 is required
ERROR: lazy loading failed for package ‘ggstatsplot’
* removing ‘/Users/labepara/Library/R/3.5/library/ggstatsplot’
Error in i.p(...) : 
  (converti depuis l'avis) installation of package ‘/var/folders/_q/fptlnf1d7rz7dyfvqtj611100000gn/T//Rtmpf9EQfU/file22617e81652/ggstatsplot_0.0.10.9000.tar.gz’ had non-zero exit status`

**Comment by lburleigh** [comment]
> This is a new function still under development and not present in the `CRAN` version. You will, therefore, have to download the development version of the package with the following command-
> 
> ```
> remotes::install_github(repo = "IndrajeetPatil/ggstatsplot")
> ```

I ran the download using this command and all went fine but I'm still getting this same error when trying to run the examples using ggwithinstats! Any thoughts?

**Comment by lburleigh** [comment]
> This is a new function still under development and not present in the `CRAN` version. You will, therefore, have to download the development version of the package with the following command-
> 
> ```
> remotes::install_github(repo = "IndrajeetPatil/ggstatsplot")
> ```

I ran the download using this line and all went fine but I'm still getting the same error when trying to run the examples using ggwithinstats! Any thoughts?

**Comment by IndrajeetPatil** [comment]
Can you refresh your R session and run the following code and paste the output from the terminal in this thread?

```r
install.packages("sessioninfo")
library(ggstatsplot)
sessioninfo::session_info(include_base = TRUE)
```

**Comment by lburleigh** [comment]
Nevermind! I forgot I tried to take a shortcut and use the time constraint download options. Once I installed using the longer download below it works. Thank you for the quick response! 

`remotes::install_github(
  repo = "IndrajeetPatil/ggstatsplot", # package path on GitHub
  dependencies = TRUE, # installs packages which ggstatsplot depends on
  upgrade_dependencies = TRUE # updates any out of date dependencies
)`
